pkg/utils/clusterinfo/clusterinfo.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clusterinfo
+
+type ServerStatus uint
+
+const (
+	Up        ServerStatus = 0
+	Offline   ServerStatus = 1
+	Tombstone ServerStatus = 2
+	Unknown   ServerStatus = 3
+)
+
+// ServerVersionInfo is the server version and git_hash.
+type ServerVersionInfo struct {
```

Comment:
Please call it `ComponentVersionInfo`. `Server` will cause misunderstanding for the Dashboard Server.